Sources/MongoSwift/MongoError.swift

```diff
@@ -278,6 +278,22 @@ private func getBulkWriteErrorFromReply(
     )
 }
 
+/// Internal function used by write methods performing single writes that are implemented via the bulk API. If the
+/// input error is a ServerError.bulkWriteError, converts it to a ServerError.writeError. Otherwise, returns the
+/// error as is.
+internal func convertBulkWriteError(_ error: Error) -> Error {
+    if case let ServerError.bulkWriteError(bulkWriteErrors, writeConcernError, _, errorLabels) = error {
```

Comment:
I always forget this `if case let` syntax is a thing, first time I've actually used it 😱

Reply:
🤘